### DISCUSSION TITLE
Reset Button

### DISCUSSION THREAD

**Discussion by wigandt-dot** [CMPUT301F21T22/Nabu#14] [Ideas]
I was creating a few CRC cards for the fragments and was wondering on whether or not we should include a button to "reset" the app. Upon clicking it the user might get a pop-up doing the whole: "are you sure you want to delete all of your habits?" If they accept, then all of their current data would be deleted, except for the account itself. Any thoughts?

**Comment by trnglina** [comment]
This seems like a really useful debug feature. We can add it to the (currently fairly empty) settings menu.
